modules/logging-release-notes-5-8-5.adoc

```diff
@@ -0,0 +1,41 @@
+This release includes link:https://access.redhat.com/errata/RHBA-2024:1474[OpenShift Logging Bug Fix Release 5.8.5].
+
+[id="logging-release-notes-5-8-5-bug-fixes"]
+== Bug fixes
+
+* Before this update, the configuration of the `ServiceMonitor` by Loki Operator could match many Kubernetes services, which led to Loki Operator's metrics being collected multiple times. With this update, the `ServiceMonitor` setup now only matches the dedicated metrics service. (link:https://issues.redhat.com/browse/LOG-5250[LOG-5250])
+
+
+* Before this update, the Red Hat build pipeline didn't use the existing build details in Loki builds, leaving out information like revision, branch, and version. With this update, the Red Hat build pipeline now adds these inputs to the Loki builds, fixing the issue. (link:https://issues.redhat.com/browse/LOG-5201[LOG-5201])
+
+
+* Before this update, the Loki Operator checked whether the pods were running to decide if the `LokiStack` was ready. With this update, it also checks if the pods are ready, so the readiness of the `LokiStack` reflects the state of its components. (link:https://issues.redhat.com/browse/LOG-5171[LOG-5171])
+
+
+* Before this update, executing a query for logs metrics caused an error on the histogram. With this update, the histogram toggle and chart are disabled and hidden because the histogram doesn't work with log metrics. (link:https://issues.redhat.com/browse/LOG-5044[LOG-5044])
+
+
+* Before this update, the Loki and Elasticsearch bundle had the wrong `maxOpenShiftVersion`, triggering `IncompatibleOperatorsInstalled` alerts. With this update, including 4.16 as the `maxOpenShiftVersion` property in the bundle fixes the issue. (link:https://issues.redhat.com/browse/LOG-5272[LOG-5272])
+
+
+* Before this update, the build pipeline did not include linker flags for the build date, causing Loki builds to show empty strings for `buildDate` and `goVersion`. With this update, adding the missing linker flags in the build pipeline fixes the issue. (link:https://issues.redhat.com/browse/LOG-5274[LOG-5274])
+
+
+* Before this update, a bug in LogQL parsing left out some line filters from the query. With this update, the parsing now includes all the line filters while keeping the original query unchanged. (link:https://issues.redhat.com/browse/LOG-5270[LOG-5270])
+
+* Before this update, the Loki operator ServiceMonitor in openshift-operators-redhat used static token and CA files for authentication, causing errors in the Prometheus operator in User-Workload-Monitoring on such ServiceMonitor configuration. With this update, the Loki operator ServiceMonitor in openshift-operators-redhat now references a serviceaccount token secret via a LocalReference. This approach resolves the issue, allowing the User-Workload-Monitoring Prometheus operator to handle the Loki operator ServiceMonitor successfully, enabling Prometheus to scrape the Loki operator metrics. (link:https://issues.redhat.com/browse/LOG-5240[LOG-5240])
```

Comment:
```suggestion
* Before this update, the Loki Operator `ServiceMonitor` in `openshift-operators-redhat` used static token and CA files for authentication, which caused errors in the Prometheus Operator in user workload monitoring with such a `ServiceMonitor` configuration. With this update, the Loki Operator `ServiceMonitor` in `openshift-operators-redhat` now references a serviceaccount token secret through a `LocalReference`. This approach fixes the issue and enables the Prometheus Operator user workload monitoring to successfully handle the Loki Operator `ServiceMonitor` so that Prometheus can retrieve the Loki Operator metrics. (link:https://issues.redhat.com/browse/LOG-5240[LOG-5240])
```